homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" type="text/css" href="stylesheet.css">
+        <title>About</title>
+    </head>
+    <body>
+        <header></header>
+        <main>
+            <h1>About me</h1>
+            <p class="main__content">Hi! My name is Vladyslav Brusentsov and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system. 
+                This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it">'Frontend for beginners' from Masters Academy in 2023.</a>
+                This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio. 
+                You can contact me via [Links to Socials Nets] and/or check out my GitHub.
```

Comment:
You have to change layout like on design

Reply:
Done